### PR TITLE
Add Style enforcer (via Rubocop)

### PR DESCRIPTION
**Pull request by bf4**
It will fail the build, but as it is currently, most of the cops are
'todos'. Great for new contributors.. :)

Style/StringLiterals single quote all the things

### CODE REVIEW

**Comment by beauby** [comment]
:+1:

**Comment by bf4** [comment]
There interest in this?

File: lib/generators/serializer/serializer_generator.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-      argument :attributes, :type => :array, :default => [], :banner => "field:type field:type"
+      argument :attributes, :type => :array, :default => [], :banner => 'field:type field:type'
 
-      class_option :parent, :type => :string, :desc => "The parent class for the generated serializer"
+      class_option :parent, :type => :string, :desc => 'The parent class for the generated serializer'
 
       def create_serializer_file
-        template 'serializer.rb', File.join('app/serializers', class_path, "#{file_name}_serializer.rb")
+        template 'serializer.rb.erb', File.join('app/serializers', class_path, "#{file_name}_serializer.rb")
```
Comment (joaomdmoura) [inline lib/generators/serializer/serializer_generator.rb]: :+1:

**Comment by joaomdmoura** [comment]
Never used `Rubocop` but it seems really nice. This will help into get new ppl to contribute to it :smile:.
Only got scared with the _todo.yml file. Binding it to travis was nice as well!

**Pull request merged**